test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -31,7 +31,8 @@ LOG_DIR=${HOME_DIR}/logs
 
 echo "Simply verify if smdebug is installed correctly."
 echo "Cloning git repo for release tag $SMDEBUG_RELEASE_TAG"
-git clone -b ${SMDEBUG_RELEASE_TAG} https://github.com/awslabs/sagemaker-debugger.git sagemaker-debugger || exit 1
+# temprarily using master
```

Comment:
Can we add a comment saying that this change will be reverted in the next release of smdebug?